modules/OSSMail/models/Record.php

```diff
@@ -95,8 +95,14 @@ public static function imapConnect($user, $password, $host = false, $folder = 'I
 		}
 
 		imap_timeout(IMAP_OPENTIMEOUT, 5);
-		\App\Log::trace("imap_open({" . $host . ":" . $port . "/imap" . $sslMode . $validatecert . "}$folder, $user , $password) method ...");
-		$mbox = @imap_open("{" . $host . ":" . $port . "/imap" . $sslMode . $validatecert . "}$folder", $user, $password);
+		$options = 0;
+		$n_retries = 0;
+		$params = null;
+		if ($rcConfig['imap_fix_msexchange_kerberos']) {
+			$params = array('DISABLE_AUTHENTICATOR' => 'GSSAPI');
```

Comment:
SMTP is not affected (caused by php_imap).

Roundcube uses own imap implementation, not php_imap.